ethexe/common/src/db.rs

```diff
@@ -31,8 +32,7 @@ use gear_core::{
 use gprimitives::H256;
 
 pub trait BlockMetaStorageRead {
-    fn block_prepared(&self, block_hash: H256) -> bool;
-    fn block_computed(&self, block_hash: H256) -> bool;
+    fn block_meta(&self, block_hash: H256) -> BlockMeta;
```

Comment:
add comments for read and write that if item doesn't exist in storage - it will return default